R/convertFromBIOM.R

```diff
@@ -292,7 +297,7 @@ convertFromBIOM <- function(
     # If only one prefix was found (like it should be), get the corresponding
     # rank name.
     if( sum(found_rank) == 1 ){
-        colname <- TAXONOMY_RANKS[found_rank]
+        colname <- names(prefixes)[found_rank]
```

Comment:
There was a bug that is now fixed. Colnames were set based on index of listed prefix and TAXONOMY_RANKS but they did not match (there were domain prefix that was not in TAXONOMY_RANKS).